### PR TITLE
Bumped dependencies

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,13 +26,13 @@
         "symfony/config": "^3.4 || ^4.0",
         "symfony/translation": "^3.4 || ^4.0",
         "twig/extensions": "^1.5",
-        "twig/twig": "^2.0"
+        "twig/twig": "^2.9"
```
Comment (greg0ire) [inline composer.json]: Probably not needed?

Comment (kunicmarko20) [reply composer.json]: it was released 3 days ago

**Comment by greg0ire** [comment]
Should be retargeted to the new 1.x, right?

File: composer.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
         "sonata-project/core-bundle": ">=3.13"
     },
     "require-dev": {
-        "jms/serializer-bundle": "^1.0 || ^2.0",
+        "jms/serializer-bundle": "^3.3",
```
Comment (kunicmarko20) [inline composer.json]: can we just drop this on a stable branch? there should be a big difference between 2.0 and 3.0

Comment (core23) [reply composer.json]: It's a dev dependency....

**Pull request merged**